test/functional/Main.hs

```diff
@@ -20,22 +21,25 @@ import Progress
 import Reference
 import Rename
 import Symbol
+import System.Environment
 import TypeDefinition
 
 main :: IO ()
-main =
+main = do
+    setEnv "TASTY_NUM_THREADS" "1"
```

Comment:
Could you put some comments that explain why do you add this env var?

Reply:
I don't think it was strictly necessary, so I have reverted it instead